packages/agreement-process/src/utilities/errorMappers.ts

```diff
@@ -185,3 +185,16 @@ export const computeAgreementsStateErrorMapper = (
   match(error.code)
     .with("badRequestError", () => HTTP_STATUS_BAD_REQUEST)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const verifyTenantCertifiedAttributesErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("tenantNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with(
+      "eServiceNotFound",
+      "descriptorNotFound",
+      () => HTTP_STATUS_BAD_REQUEST
```

Comment:
shouldn't be a not found?

Reply:
> shouldn't be a not found?
In these cases we usually decide which is the main "subject" of the route, on which we return 404 if not found (in this case the tenant), meanwhile the other objects if not found we return a 400 since it's an incorrect input.

We can rethink this behaviour if we want, but for now it's of the same pattern in various mappers

Reply:
Ok to leave it like this also for me, when this route is called we shall have eservice and descriptor defined and found, tenant is the main subject (the potential consumer, it can be any tenant) - the case where they are not should not be possible, so I think it's ok to return a bad request and not 404